developer_defaults.tf

```diff
@@ -145,11 +145,6 @@ variable "backup_retention_period" {
   default     = 0
 }
 
-variable "RETENTION_DAYS" {
```

Comment:
You should still declare the variable here as a developer default of 1? and pass it into `module "survey-runner-queue" {` in `survey-runner.tf`